jvm/src/test/scala/BaseUnpicklingSuite.scala

```diff
@@ -0,0 +1,21 @@
+import tastyquery.Contexts
+import tastyquery.Contexts.Context
+import tastyquery.ast.Trees.Tree
+import tastyquery.ast.Types.Type
+import tastyquery.reader.TastyUnpickler
+
+import java.nio.file.{Files, Paths}
+
+abstract class BaseUnpicklingSuite extends munit.FunSuite {
+  val ResourceProperty = "test-resources"
+
+  def unpickle(filename: String)(using ctx: Context = Contexts.empty): Tree = {
+    val resourcePath = getResourcePath(filename)
+    val bytes        = Files.readAllBytes(Paths.get(resourcePath))
+    val unpickler    = new TastyUnpickler(bytes)
+    unpickler.unpickle(new TastyUnpickler.TreeSectionUnpickler()).get.unpickle (using ctx).head
```

Comment:
did scalafmt add this space?

Reply:
Yes, with the dotty runner. Just updated the runner to scala3 and it's all pretty now!

Reply:
nicee